amd_openvx/openvx/ago/ago_util.cpp

```diff
@@ -870,11 +957,12 @@ void agoGetDescriptionFromData(AgoContext * acontext, char * desc, AgoData * dat
 		sprintf(desc + strlen(desc), "lut%s:%s," VX_FMT_SIZE "", virt, agoEnum2Name(data->u.lut.type), data->u.lut.count);
 	}
 	else if (data->ref.type == VX_TYPE_THRESHOLD) {
-		sprintf(desc + strlen(desc), "threshold%s:%s,%s", virt, agoEnum2Name(data->u.thr.thresh_type), agoEnum2Name(data->u.thr.data_type));
-		if (data->u.thr.thresh_type == VX_THRESHOLD_TYPE_BINARY)
+		sprintf(desc + strlen(desc), "threshold%s:%s,%u,%u", virt, agoEnum2Name(data->u.thr.thresh_type), data->u.thr.input_format, data->u.thr.output_format);
```

Comment:
Codacy found an issue: [Does not handle strings that are not \0-terminated; if given one it may perform an over-read (it could cause a crash if unprotected) (CWE-126).  ](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5686893)